src/openforms/api/tests/test_error_format.py

```diff
@@ -0,0 +1,172 @@
+from django.utils.translation import gettext as _
+
+from rest_framework.test import APIRequestFactory, APITestCase
+
+from . import error_views as views
+
+
+class ExceptionHandlerTests(APITestCase):
+    """
+    Test the error handling responses
+    """
+
+    maxDiff = None
+    factory = APIRequestFactory()
+
+    def assertErrorResponse(self, view, expected_data: dict):
+        _view = view.as_view()
+        # method doesn't matter since we're using `dispatch`
+        request = self.factory.get("/some/irrelevant/url")
+
+        response = _view(request)
+
+        expected_status = expected_data["status"]
+        self.assertEqual(response.status_code, expected_status)
+        self.assertEqual(response["Content-Type"], "application/problem+json")
+
+        # can't verify UUID...
+        self.assertTrue(response.data["instance"].startswith("urn:uuid:"))
+        del response.data["instance"]
+
+        exc_class = view.exception.__class__.__name__
+        expected_data["type"] = f"http://testserver/fouten/{exc_class}/"
+        self.assertEqual(response.data, expected_data)
+
+    def test_400_error(self):
+        self.assertErrorResponse(
+            views.ValidationErrorView,
+            {
+                "code": "invalid",
+                "title": _("Invalid input."),
+                "status": 400,
+                "detail": "",
+                "invalid_params": [
+                    {
+                        "name": "foo",
+                        "code": "validation-error",
+                        "reason": _("Invalid data."),
+                    }
+                ],
+            },
+        )
+
+    def test_401_error(self):
+        self.assertErrorResponse(
+            views.NotAuthenticatedView,
+            {
+                "code": "not_authenticated",
+                "title": "Authenticatiegegevens zijn niet opgegeven.",
```

Comment:
Please ensure _every_ test uses translatable strings _everywhere_ for the user-facing messages. I'm not going to copy paste the same comment for every individual test.